tests/components/uptime/test_sensor.py

```diff
@@ -1,36 +1,35 @@
 """The tests for the uptime sensor platform."""
 import pytest
+from syrupy.assertion import SnapshotAssertion
+from syrupy.filters import props
 
-from homeassistant.components.sensor import SensorDeviceClass
 from homeassistant.components.uptime.const import DOMAIN
-from homeassistant.const import ATTR_DEVICE_CLASS
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import device_registry as dr, entity_registry as er
 
 from tests.common import MockConfigEntry
 
 
+@pytest.mark.usefixtures("init_integration")
 @pytest.mark.freeze_time("2022-03-01 00:00:00+00:00")
 async def test_uptime_sensor(
     hass: HomeAssistant,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
     init_integration: MockConfigEntry,
+    snapshot: SnapshotAssertion,
 ) -> None:
     """Test Uptime sensor."""
-    state = hass.states.get("sensor.uptime")
-    assert state
+
+    assert (state := hass.states.get("sensor.uptime"))
```

Comment:
why the walrus?

Reply:
To catch the None state in a simple assertion.
Compared to a snapshot the next step could be done directly, but this (at least for me) feels like a sane check before comparing a larger set of states.